src/rgw/rgw_auth.h

```diff
@@ -455,7 +455,7 @@ class LocalApplier : public IdentityApplier {
 
   LocalApplier(CephContext* const cct,
                const RGWUserInfo& user_info,
-               std::string subuser)
+              const std::string& subuser)
```

Comment:
I'm afraid that taking `subuser` by `const &` here isn't the best option. It would make moving internal data of the incoming `std::string` to the `subuser` member impossible. Please consider a scenario when someone instantiate `LocalApplier` with a temporal variable as the `subuser`. It can perfectly bind `const &` but `std::move` will result in `const &&` in such case, I guess.